site/README.md

```diff
@@ -0,0 +1,54 @@
+# aip.dev static site generator
+
+Welcome to our NIH static site generator.
+
+**Note:** If you do not care about this and just want to write or modify AIPs,
+see `CONTRIBUTING.md`.
+
+## Why?
+
+GitHub Pages normally automatically builds documentation with [Jekyll][], but
+as the AIP system has grown, we are beginning to reach the limits of what
+Jekyll can handle:
+
+- AIP adoption is handled through fork-and-merge and top-down configuration
+  files will lead to repetitive merge conflicts.
+- Our grouping and listing logic has grown complicated, and had to be
+  maintained using complex and error-prone Liquid templates.
```

Comment:
Could we have resolved this by writing tests four our liquid templates?

Reply:
I am not sure how to do that. (Perhaps it could be done though?)
Quick Google searching does not provide an obvious answer.